container/container.go

```diff
@@ -261,10 +261,16 @@ func (w *ContainerWorker) finalizeDeployment(agreementId string, deployment *con
 
 		var logConfig docker.LogConfig
 
+		// Use syslog log driver by default
+		logDriver := "syslog"
+		if service.LogDriver != "" {
+			logDriver = service.LogDriver
+		}
+
 		if !deployment.ServicePattern.IsShared("singleton", serviceName) {
 			labels[LABEL_PREFIX+".agreement_id"] = agreementId
 			logConfig = docker.LogConfig{
-				Type: "syslog",
+				Type: logDriver,
 				Config: map[string]string{
 					"tag": fmt.Sprintf("workload-%v_%v", strings.ToLower(agreementId), serviceName),
```

Comment:
I got this error when the log_driver is set to 'local' `API error (400): unknown log opt 'tag' for log driver local.`

Reply:
Yeah, as it turns out, some logging drivers doesn't support tags, and some of them don't like unknown options (json-file for example is just ignoring them). I've added removing of "tag" config for unsupported drivers (this looks simpler than adding few new new `if` blocks for preventing adding of "tag")